Week-2/2.2/src/index.css

```diff
@@ -0,0 +1,84 @@
+.box {
+    width: 100px;
+    height: 100px;
+}
+
+.red {
+    transform-origin: 0% 0%;
+    background-color: red;
+    animation-name: move;
```

Comment:
This animation supposed to move on hover, I am unable to figure out the trigger point.